src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs

```diff
@@ -1,13 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 //
-using Microsoft.Extensions.Configuration.AzureAppConfiguration;
-using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
```

Comment:
Looks like there are some unintentional whitespace changes in this file. Could you reset the whitespaces and update only the method summary?

Reply:
I intended to do that. Because I see inconsistency in line endings, some lines are LF and some CRLF. Do you suggest that I don't touch them in this PR?

part of `git diff -R` results as below:

```diff
diff --git b/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs a/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs
index 3dabf4c..657512e 100644
--- b/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs
+++ a/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 //
-using Microsoft.Extensions.Configuration.AzureAppConfiguration;
-using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;^M
+using Microsoft.Extensions.DependencyInjection;^M
 using System;
-using System.Collections.Generic;
-
+using System.Collections.Generic;^M
+^M
 namespace Microsoft.Extensions.Configuration
-{
+{^M
```

Reply:
I prefer to make it consistent, and also we won't see the warning in VS as below:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/2351748/234732506-f40bac21-3516-4f6a-bb12-5ac0611a0525.png">

BTW I glanced at other files, their EOL are all CRLF. Only `AzureAppConfigurationExtensions.cs` has mixed EOLs. So probably I should update it to CRLF instead of LF. Never mind, it's a linting issue and I can create a separate PR to do that if necessary.

I'll follow your suggestion don't touch it here. WDYT?

Reply:
I think it's a good idea to update it in a different PR. This looks good now. Thanks!